src/services/login.service.ts

```diff
@@ -0,0 +1,22 @@
+import { HttpClient  } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class LoginService {
+  constructor (private http: HttpClient) {}
+
+  loggedIn = false;
+
+  isAuthenticated () {
+    return this.loggedIn;
+  }
+
+  login (data) {
+    return new Promise(resolve => {
+      setTimeout(() => {
+        this.loggedIn = true;
```

Comment:
Maybe it would be better to use sessionStorage to avoid logout after reloading the page?

Reply:
yep, agree. I will implement this after integration with firebase